aws/resource_aws_cloudfront_distribution.go

```diff
@@ -35,18 +35,17 @@ func resourceAwsCloudFrontDistribution() *schema.Resource {
 				Set:      aliasesHash,
 			},
 			"cache_behavior": {
-				Type:     schema.TypeSet,
+				Type:     schema.TypeList,
```

Comment:
There is also a Schema migration to create here, as for security groups:
- [Resource](https://github.com/terraform-providers/terraform-provider-aws/blob/b8d4e1570fc43a2acee6b6e47f63c9db6b067fa2/aws/resource_aws_security_group_rule.go#L26)
- [Related migration](https://github.com/terraform-providers/terraform-provider-aws/blob/b8d4e1570fc43a2acee6b6e47f63c9db6b067fa2/aws/resource_aws_security_group_rule_migrate.go)

This would allow to not break things for users, and make the change invisible :)